app/server/src/client/preview/render.ts

```diff
@@ -0,0 +1,61 @@
+import { document, fetch, Node } from 'global';
+import dedent from 'ts-dedent';
+import { RenderMainArgs, FetchStoryHtmlType } from './types';
+
+const rootElement = document.getElementById('root');
+
+let fetchStoryHtml: FetchStoryHtmlType = async (url, id, params) => {
```

Comment:
```suggestion
let fetchStoryHtml: FetchStoryHtmlType = async (url, path, params) => {
```

Let's call this path and allow it to be empty (not sure if we should drop the trailing slash in that case?